tests/test_cases/test_vhdl_zerovector/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Simulator (modelsim is still 32-bit only)
+SIM ?= modelsim
+ifneq ($(filter $(SIM), modelsim),)
+    ARCH ?= i686
+endif
```

Comment:
Do I need to ad some kind of protection against simulators without VHDL?

Reply:
Because many of them have both VHDL and Verilog top levels. I'll give you some more feedback tonight once I have some time. I just wanted to ensure this wasn't merged just yet.

Reply:
Thank you. Looking forward to see you inputs tomorrow (it's getting late in the night for me)